### DISCUSSION TITLE
I'm not able to apply for github students developer pack

### DISCUSSION THREAD

**Discussion by kanishka-G** [community/community#42027] [GitHub Education]
it says me to upload a image of my university ID card which mentions my duration of studies in it. I upload that and submit but then it says me to update my profile by putting in pictures , name and bio, which I also do. But even then I'm not able to apply. can someone help me out in this. Thanks in advance

**Comment by dudkinox** [comment]
1. Make sure that you have filled out all required fields in the form or application. Sometimes there will be specific fields that are marked as "required" or "mandatory," and you will need to enter information into these fields in order to submit the form.

2. Check for any error messages or notifications that might provide more information about the problem. Sometimes there will be specific errors or issues that are preventing you from submitting the form or completing the registration process.

3. Try refreshing the page or clearing your browser's cache and cookies. Sometimes, a temporary glitch or issue with the website can cause problems with submissions or registration processes.

4. If you are still having trouble, you might try reaching out to the website or platform's customer support team for assistance. They should be able to help you troubleshoot the problem and get you back on track.

I hope these suggestions are helpful! Let me know if you have any other questions.

**Reply by kanishka-G** [reply]
Thanks a lot , @dudkinox  , how to reach the customer support team? I'm not able to see anything as such.

**Comment by samus-aran** [comment]
Hi @kanishka-G 👋 

We're happy you're here. Let's see how we can assist. Often this common pre-processing warning is displayed because the application can detect differences between your profile information and the evidence you've supplied. As you can imagine GitHub processes a number of applications - we do request that students can demonstrate that their GitHub account corresponds with the evidence given. The verification looks at your GitHub username, email and profile name (and other aspects). Do these match? Another tip is to try and not use a nickname in your profile info whilst being verified.

Hope this helps. Let us know how you get on 👍
